src/config.cpp

```diff
@@ -868,6 +869,7 @@ class SimulationConfig::Parser
         parseOptional(*runIt, "ionchannel_seed", result.ionchannelSeed, {0});
         parseOptional(*runIt, "minis_seed", result.minisSeed, {0});
         parseOptional(*runIt, "synapse_seed", result.synapseSeed, {0});
+        parseOptional(*runIt, "electrodes_file", result.electrodesFile, {""});
```

Comment:
I think it is safer to parse it to be an absolute path here, like the other file paths in the config file.

Reply:
The optional files are parsed in the same way. For example:

`parseOptional(*outputIt, "log_file", result.logFile, {""});
 parseOptional(*outputIt, "spikes_file", result.spikesFile, {"out.h5"});`